deeppavlov/requirements/syntax_parser.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/andersjo/dependency_decoding
```

Comment:
Make it an exact commit, please. So that everything would still work even if the repository is updated